dockerfile_parse/parser.py

```diff
@@ -597,11 +597,13 @@ def add_lines(self, *lines, **kwargs):
         :param all_stages: bool for whether to add in all stages for a multistage build
                            or (by default) only the last.
         :param at_start: adds at the beginning (after FROM) of the stage(s) instead of the end.
+        :param skip_scratch: skip for FROM scratch
```

Comment:
Would be nice to make this slightly more explicit, like: `skip stages which use "FROM scratch"`

Reply:
ok